### PR TITLE
ac: add Korean translation

### PR DESCRIPTION
**Pull request by korECM**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.ko/common/ac.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+`ac`
+
+- 모든 사용자의 연결된 시간을 시간 단위로 출력:
+
+`ac -p`
+
+- 특정 사용자의 연결된 시간을 시간 단위로 출력:
+
+`ac -p {{사용자 이름}}`
```
Comment (kbdharun) [inline pages.ko/common/ac.md:16]:
```suggestion
`ac -p {{사용자 명}}`
```

Ref.

https://github.com/tldr-pages/tldr/blob/1dae6c83abc376d83fff9bd61e75bc80f7000854/contributing-guides/translation-templates/common-arguments.md?plain=1#L23

File: pages.ko/common/ac.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- 특정 사용자의 연결된 시간을 평균 시간과 총 접속 시간을 일 단위로 출력:
+
+`ac -dp {{사용자 이름}}`
```
Comment (kbdharun) [inline pages.ko/common/ac.md:20]:
```suggestion
`ac -dp {{사용자 명}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Comment by korECM** [comment]
@kbdharun 
I have accepted all of the suggestions you provided! Thank you as always!

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**